monosi/config/project.py

```diff
@@ -13,6 +13,19 @@ class ProjectConfigurationDefaults:
     source_name: Optional[str] = None
     monitor_paths: List[str] = field(default_factory=lambda: ['./monitors'])
     reporter: Reporter = field(default_factory=Reporter)
+    
+    @classmethod
+    def _retrieve_project_config_path(cls, root_path):
+        root_path = os.path.normpath(root_path)
+
+        config_path_selector = os.path.join(root_path, 'monosi_project.y*ml')
```

Comment:
is the `*` to accept `yml` or `yaml`? Would `yzml` work then too? Maybe we should just restrict it to one file type

Reply:
both work here, and should not be restricted